quickjs.c

```diff
@@ -60,6 +60,12 @@
 #define CONFIG_PRINTF_RNDN
 #endif
 
+#if defined(__NEWLIB__)
+/* undefine if `struct tm` does not contain `tm_gmtoff` property */
```

Comment:
This reads weird. If we have auto-detect why do we need to undef here?

Reply:
Auto-detect was removed as per Ben's comment. Instead it's an explicit check for newlib. I can just delete this comment if you prefer.

Reply:
But if we have this then we don't need the CMake bits at all.

Reply:
4c4d73dd325935d1c6699a7b1a8024e0c1644e0f